lib/magic_test/engine.rb

```diff
@@ -14,7 +14,11 @@ class Engine < Rails::Engine
           ActionDispatch::IntegrationTest.include MagicTest::Support
         end
 
-
+        if defined? RSpec
+          RSpec.configure do |config|
+            config.include MagicTest::Support
```

Comment:
@andrewculver working for me, though I would suggest this is limited to system specs only, both to avoid slowing down/polluting other tests and because this gem is only meant to be used for system tests.  It also may make it easier to address issues like this: https://github.com/bullet-train-co/magic_test/issues/5
```suggestion
            config.include MagicTest::Support, type: :system
```

See RSpec's docs on the `type: :system` designation [here.](https://relishapp.com/rspec/rspec-rails/docs/system-specs/system-spec)